src/MyPlot/provider/MySQLProvider.php

```diff
@@ -202,6 +227,14 @@ public function getNextFreePlot(string $levelName, int $limitXZ = 0) {
 		return null;
 	}
 
+	public function close() : void {
+		$this->db->close();
+		$this->plugin->getLogger()->debug("MySQL database closed!");
+	}
+	
+	/**
+	 * @return bool
+	 */
 	private function reconnect() : bool {
 		if(!$this->db->ping()) {
 			$this->plugin->getLogger()->error("The MySQL server can not be reached! Trying to reconnect!");
```

Comment:
Translations?

Reply:
only messages sent to clients get translations

Reply:
How does pmmp translate their internal debugs/warnings then?

Reply:
I don't know what pmmp's method is, but I'm not adding translations for console outputs. It's not needed